scripts/msct_image.py

```diff
@@ -659,6 +658,19 @@ def get_permutation_from_orientations(orientation_in, orientation_out):
 
         return perm, inversion
 
+    def get_orientation_3d(self):
+        """Returns the orientation of an image.
+
+        Uses the binary implementation of `get_orientation`. Recommend using
+        `get_orientation` instead and deprecate this function.
+        """
+        status, output = sct.run("isct_orientation3d -i %s -get" % self.absolutepath, 0)
```

Comment:
Just noticed that now, sorry: if you can use a list instead of a string for sct.run, it will be consistent with other calls.